website/content/docs/agent/caching/persistent-caches/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: docs
+page_title: Vault Agent Persistent Cache Types
+sidebar_title: Persistent Cache
+description: Persistent Cache Types for Vault Agent Caching
+---
+
+# Vault Agent Persistent Cache Types
+
+Please see the sidebar for available types and their usage/configuration.
```

Comment:
I forget, is this index/overview page required by the templating system? Otherwise it seems like we could drop it.

Reply:
They're required else it throws an error.